Account/FxAClient10.swift

```diff
@@ -48,6 +60,37 @@ public struct FxASignResponse {
     }
 }
 
+public struct FxAStatusResponse {
+    let exists: Bool
+    let locked: Bool
+
+    init(exists: Bool, locked: Bool) {
+        self.exists = exists
+        self.locked = locked
+    }
+}
+
+public struct FxADevicesResponse {
+    struct FxADevice {
```

Comment:
Question: Any need for this to be defined within the scope of the response struct? Since the `devices` property is non-private, the type will be used outside of this struct. Seems annoying to have to always preface it with `FxADevicesResponse.FxADevice` if we don't need to.